onnxruntime/core/framework/session_options.h

```diff
@@ -82,6 +82,9 @@ struct SessionOptions {
   // unless the filepath ends in '.ort' (case insensitive).
   std::basic_string<ORTCHAR_T> optimized_model_filepath;
 
+  // external data folder path if the model is loaded from memory buffer
+  std::basic_string<ORTCHAR_T> external_data_path;
```

Comment:
isn't it better to add new fields at end of struct rather than middle?

Reply:
this is an internal header isn't it, so the concerns about ordering which apply to the public API might not be relevant here?

Reply:
yes, true. only caught my attention because it broke some training code that was using the internal struct. hopefully there aren't other obscure cases that aren't covered by CI's.